service/src/main/java/org/cbioportal/genome_nexus/service/remote/MyVariantInfoDataFetcher.java

```diff
@@ -0,0 +1,63 @@
+package org.cbioportal.genome_nexus.service.remote;
+
+import org.cbioportal.genome_nexus.model.my_variant_info_model.MyVariantInfo;
+import org.cbioportal.genome_nexus.service.exception.ResourceMappingException;
+import org.cbioportal.genome_nexus.service.transformer.ExternalResourceTransformer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.ResourceAccessException;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+import java.util.Map;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+
+@Component
+public class MyVariantInfoDataFetcher extends BaseExternalResourceFetcher<MyVariantInfo>
+{
+    private static final String MAIN_QUERY_PARAM = "variant";
+    private static final String PLACEHOLDER = "VARIANT";
+
+    private final ExternalResourceTransformer<MyVariantInfo> transformer;
+
+    @Autowired
+    public MyVariantInfoDataFetcher(ExternalResourceTransformer<MyVariantInfo> transformer,
+                                       @Value("${myvariantinfo.url}") String myVariantInfoUrl)
```

Comment:
@emz1003 let's put a default here as well in case you forget to specify them in the properties:

```
${myvariantinfo.url:http://myvariant.info/v1/variant/VARIANT}
```